api/v1alpha1/cdapmaster_types.go

```diff
@@ -124,6 +124,8 @@ type CDAPServiceSpec struct {
 	SecretVolumes map[string]string `json:"secretVolumes,omitempty"`
 	// SecurityContext overrides the security context for the service pods.
 	SecurityContext *SecurityContext `json:"securityContext,omitempty"`
+	// Lifecycle is to specify Container Lifecycle hooks provided by Kubernetes for containers
```

Comment:
Please add a comment specifying how these lifecycle hooks are applied to containers, e.g. are they applied to all containers (including init containers)? Or only specific containers?

Reply:
Updated.